typescript/web/src/components/members/new-member-modal.tsx

```diff
@@ -59,7 +59,10 @@ export const NewMemberModal = ({
     const inputValue = e.target.value;
     setValue(inputValue);
   };
-  const emails = value.split("\n").filter((email) => email !== "");
+  const emails = value
+    .split("\n")
+    .filter((email) => email !== "")
+    .map((email) => email.trim());
```

Comment:
The map should be before the filter so that values with just spaces will get removed.

Reply:
Done